lib/roby/exceptions.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "pp"
```

Comment:
Forgot?

Reply:
No. On purpose. PP is actually used in the file and in some corner cases this file would be loaded directly. This led to error reporting to ... error.